src/app/lib/lnd-http/types.ts

```diff
@@ -151,6 +189,9 @@ export interface GetNodeInfoResponse {
 
 export interface GetChannelsResponse {
   channels: Channel[];
+  pending_force_closing_channels: PendingForceClosingChannel[];
+  waiting_close_channels: WaitingCloseChannel[];
+  pending_open_channels: PendingOpenChannel[];
 }
```

Comment:
```typescript
export interface GetPendingChannelsResponse {
  pending_force_closing_channels: Channel[];
  waiting_close_channels: Channel[];
  pending_open_channels: Channel[];
}
```

Reply:
Also, was having an issue with the channel interface because the pending channels response has another level named "channel", so I modified the channel interface as a work around.